### PR TITLE
[GSoC] Formula Editor Lists

### PR DESCRIPTION
**Pull request by obusher**
* Reopening of #1100 Pullrequest from @bernhardsp.
* Rebased to current master. 
* [Testrun (green)](https://jenkins.catrob.at/view/All-Categories/view/Catroid-multi-job/job/Catroid-Multi-Job-Custom-Branch-RELOADED/696/) :smile: 

Link to [deprecated PR](https://github.com/Catrobat/Catroid/pull/1168).

### CODE REVIEW

**Comment by obusher** [comment]
Can not add the "XML - Change" - label. :sob:

**Comment by obusher** [comment]
[Test green](https://jenkins.catrob.at/view/All-Categories/view/Catroid-multi-job/job/Catroid-Multi-Job-Custom-Branch-RELOADED/1013/) with 1 checkstyle warning which i can not reproduce locally.

File: catroid/src/org/catrobat/catroid/io/StorageHandler.java [hunk 1]
```diff
@@ -44,6 +44,7 @@
 import org.catrobat.catroid.content.WhenScript;
 import org.catrobat.catroid.content.XmlHeader;
 import org.catrobat.catroid.content.bricks.Brick;
+import org.catrobat.catroid.content.bricks.AddItemToUserListBrick;
```
Comment (robertpainsi) [inline catroid/src/org/catrobat/catroid/io/StorageHandler.java]: I guess the alphabetic order is A then B :wink:
```
import org.catrobat.catroid.content.bricks.AddItemToUserListBrick;
import org.catrobat.catroid.content.bricks.Brick;
```

**Comment by obusher** [comment]
[Test Green] (https://jenkins.catrob.at/view/All-Categories/view/Catroid-multi-job/job/Catroid-Multi-Job-Custom-Branch-RELOADED/1020/)
    Happy Merging! :)

@robertpainsi : As you mentioned in IRC, execution of the Catroid:clean task is necessary for checkstyle to output warnings to the console. :dash:

**Comment by avihric** [comment]
ok to test

**Comment by avihric** [comment]
@obusher can you rebase again please as I merged #1194

**Comment by obusher** [comment]
[Test Green] (https://jenkins.catrob.at/view/All-Categories/view/Catroid-multi-job/job/Catroid-Multi-Job-Custom-Branch-RELOADED/1027/)

**Pull request merged**